.changeset/tasty-ties-hang.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/backend-app-api': patch
+'@backstage/config-loader': patch
+'@backstage/cli': patch
+---
+
+Added flag --disable-config-watching, it allows to disable the config file watching
```

Comment:
Could you split this into 3 changesets? The current messaging only makes sense for `backend-app-api`